OracleAccessManagement/dockerfiles/12.2.1.4.0/README.md

```diff
@@ -0,0 +1,447 @@
+	
+   No output is returned to the screen.
+   
+   
+### Pulling the Oracle FMW Infrastructure 12.2.1.x image
+
+  1. Launch a browser and access the [Oracle Container Registry](https://container-registry.oracle.com/).
+  2. Click **Sign In** and login with your username and password.
+  3. In the **Search** field enter **fmw-infrastructure** and press Enter.
+  4. Click **fmw-infrastructure** Oracle Fusion Middleware Infrastructure.
+  5. In the **Terms and Conditions** box, select Language as **English**. Click **Continue** and ACCEPT "**Terms and Restrictions**".
+  6. On your Docker environment login to the Oracle Container Registry and enter your Oracle SSO username and password when prompted:
+  
+    $ docker login container-registry.oracle.com
+    Username: <username>
+    Password: <password>
+   
+   For example:
+   
+    $ docker login container-registry.oracle.com
+    Username: joe.bloggs@example.com
+    Password:
+    Login Succeeded
+  
+  7. Pull the latest fmw-infrastructure image:
+  
+    $ docker pull container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.4
+	
+   The output should look similar to the following:
+   
+    Trying to pull repository container-registry.oracle.com/middleware/fmw-infrastructure ...
+    12.2.1.4: Pulling from container-registry.oracle.com/middleware/fmw-infrastructure
+    bce8f778fef0: Pull complete
+    9adc5789a738: Pull complete
+    b80677728d7c: Pull complete
+    abcd1f688714: Pull complete
+    59760436a4c4: Pull complete
+    23a18e3f8da2: Pull complete
+    0e31e4e00dc9: Pull complete
+    Digest: sha256:f1aa70158943af2d109dd0720f8a5bd9bfb2e7778c3570520a24a8b176d86d93
+    Status: Downloaded newer image for container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.4
+    container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.4
+	
+  8. Run the `docker tag` command to tag the image as follows:
+  
+    docker tag container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.4 oracle/fmw-infrastructure:12.2.1.4.0
+	
+   No output is returned to the screen.
+
+  9. Run the `docker images` command to show the image is installed into the repository. The output should look similar to this:
+	
+	$ docker images
+	
+    REPOSITORY                                                    TAG                 IMAGE ID            CREATED             SIZE
+    container-registry.oracle.com/middleware/fmw-infrastructure   12.2.1.4            760157811a73        4 weeks ago         2.33GB
+    oracle/fmw-infrastructure                                     12.2.1.4.0          760157811a73        4 weeks ago         2.33GB
+    container-registry.oracle.com/java/serverjre                  8                   da8a8a240247        2 months ago        305MB
+    oracle/serverjre                                              8                   da8a8a240247        2 months ago        305MB
+	
+
+	
+### Downloading the OAM Docker files
```

Comment:
Pulling the GitHub repo should be first because it'll be required to build the Server JRE and FMW Infrastructure images.